data/campaigns/World_Conquest/gui/help_dialog.cfg

```diff
@@ -257,7 +257,7 @@
                                             border_size = 5
                                             [scroll_label]
                                                 id = "label_content"
-                                                label = "Text"
+                                                label = _ "Text"
```

Comment:
This too should not be translatable. It should be empty or absent.

Reply:
Replaced it with empty strings.